source/chapters/import_from_serato.rst

```diff
@@ -0,0 +1,83 @@
+.. _import-serato-metadata:
+
+Importing metadata from Serato
+*************
```

Comment:
Please have a look at the "netlify/mixxx-manual/deploy-preview" check below this PR. If you click on it, you can see the following warning:

```
9:53:50 PM: Warning, treated as error:
9:53:50 PM: /opt/build/repo/source/chapters/import_from_serato.rst:4:Title underline too short.
9:53:50 PM: Importing metadata from Serato
9:53:50 PM: *************
```

So the fix for that would be:

```suggestion
Importing metadata from Serato
******************************
```

Reply:
Thank you so much for the hint! i would amend the last commit just as @ronso0 taught me